problem_builder/public/js/student_answers_dashboard.js

```diff
@@ -0,0 +1,299 @@
+function StudentAnswersDashboardBlock(runtime, element) {
+    'use strict';
+    var $element = $(element);
+
+    // Pagination
+
+    var Result = Backbone.Model.extend({
+
+        initialize: function(attrs, options) {
+            _.each(_.zip(Result.properties, options.values), function(pair) {
+                this.set(pair[0], pair[1]);
+            }, this);
+        }
+
+    }, { properties: ['section', 'subsection', 'unit', 'type', 'question', 'answer', 'username'] });
+
+    var Results = Backbone.PageableCollection.extend({
+
+        model: Result,
+
+        getCurrentPage: function(object) {
+            var currentPage = this.state.currentPage;
+            if (object) {
+                return this.getPage(currentPage);
+            }
+            return currentPage;
+        },
+
+        getTotalPages: function() {
+            return this.state.totalPages;
+        }
+
+    });
+
+    var ResultsView = Backbone.View.extend({
+
+        render: function() {
+            this._insertRecords(this.collection.getCurrentPage(true));
+            this._updateControls();
+            this.$('#total-pages').text(this.collection.getTotalPages() || 0);
+            return this;
+        },
+
+        _insertRecords: function(records) {
+            var tbody = this.$('tbody');
+            tbody.empty();
+            records.each(function(result, index) {
+                var row = $('<tr>');
+                if (index % 2 === 0) {
+                    row.addClass('even');
```

Comment:
Agreed (I ended up using `tr:nth-child(odd)` to make sure the table layout matches the [mockup](https://tasks.opencraft.com/secure/attachment/10402/instructor_dashboard.png)).